docs/sql/statements/alter-table.rst

```diff
@@ -173,6 +174,20 @@ expression <sql-literal-value>`.
 
 It's possible to add multiple columns at once.
 
+.. _sql-alter-table-drop-column:
+
+``DROP COLUMN``
+--------------
+
+Can be used to drop a additional column from a table.
```

Comment:
why additional? maybe just "drop a column"?

Reply:
thx, was more of starting a place holder to continue writing here once the whole feature is done.